samples/cf-multi-tenant-application/README.md

```diff
@@ -0,0 +1,11 @@
+# SAP Cloud SDK for JavaScript Multi-Tenant Sample Application
+
+This code samples in this directory are a reference for the [multi-tenant application](https://sap.github.io/cloud-sdk/docs/js/tutorials/multi-tenant-application) tutorial.
+You need to make adjustments to the code before deployment described in the tutorial.
+
+Also note that the samples here are intended as a didactic example and are not necessary a best practice.
+
```

Comment:
[req] Would be nice to keep the Readme structure a bit consistent across samples. For example, ## Description section and also a ## Download and Installation for sparse checkout.

Reply:
done